examples/epstein_civil_violence_experimental/Readme.md

```diff
@@ -0,0 +1,51 @@
+# Epstein Civil Violence Model
+
+This is an experimental implementation of the Epstein Civil Violence Model, which simulates the dynamics of civil violence with agents representing citizens and cops.
+
+
+## Summary
+
+This model is based on Joshua Epstein's simulation of how civil unrest grows and is suppressed. Citizen agents wander the grid randomly, and are endowed with individual risk aversion and hardship levels; there is also a universal regime legitimacy value. There are also Cop agents, who work on behalf of the regime. Cops arrest Citizens who are actively rebelling; Citizens decide whether to rebel based on their hardship and the regime legitimacy, and their perceived probability of arrest.
+
+The model generates mass uprising as self-reinforcing processes: if enough agents are rebelling, the probability of any individual agent being arrested is reduced, making more agents more likely to join the uprising. However, the more rebelling Citizens the Cops arrest, the less likely additional agents become to join.
+
+## Installation
+
+To install the dependencies use pip and the requirements.txt in this directory. e.g.
+
+```
+    # First, we clone the Mesa repo
+    $ git clone https://github.com/projectmesa/mesa.git
+    $ cd mesa
+    # Then we cd to the example directory
+    $ cd examples/epstein_civil_violence_experimental
+    $ pip install -r requirements.txt
+```
+
+## How to Run
+
+You can run the batch model directly by executing the `app.py` file. as
+
+```
+    python app.py
```

Comment:
Have you followed these instructions yourself? Do they work as expected? If so, could you share a screenshot of how it looks?

In my experience you should run `solara run app.py`, and then the browser opens by itself.